### DISCUSSION TITLE
How can I apply lua filters to every file?

### DISCUSSION THREAD

**Discussion by IllustratedMan-code** [srid/emanote#559] [Q&A]
I'm exploring the lua filter functionality, but it seems that I can only enable lua filters for a single file. Is there any way to make it so a lua filter is applied to all files?

**Comment by srid** [comment]
@IllustratedMan-code Subscribe to https://github.com/srid/emanote/issues/721 for Lua related work.

Lua filters can run IO actions (AFAIU), so it can be expensive to run them on all files. Also, there's a question of whether to apply these filters during parse time (expensive) or during render time (cheap). Mermaid, for example, can happen at render time (#625) ... but anything that modifies wikilinks and such should happen during parse time.
